.changes/unreleased/FEATURES-20250826-135328.yaml

```diff
@@ -0,0 +1,5 @@
+kind: FEATURES
```

Comment:
hmm, not sure if this is correct. These are released features and the latest is v0.16.1